contracts/examples/TokenCreateContract.sol

```diff
@@ -0,0 +1,200 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity >=0.5.0 <0.9.0;
+pragma experimental ABIEncoderV2;
+
+import "./TokenCreate.sol";
+
+contract TokenCreateContract is TokenCreate {
```

Comment:
You should wait on https://github.com/hashgraph/hedera-smart-contracts/pull/53 to go in.
Might be some resolutions to be made between the two